dist/realtime.js

```diff
@@ -693,31 +688,52 @@
      * Triggering
      */
 
-    on: function (name, handler) {
-      if (this.__bindings.hasOwnProperty(name))
-        this.__bindings[name].push(handler);
-      else
-        this.__bindings[name] = [handler];
+    on: function (name, handler, context, once) {
+      if (context === null)
+        context = undefined;
+      if (!this.__bindings.hasOwnProperty(name))
+        this.__bindings[name] = [];
+      this.__bindings[name] = this.__bindings[name].concat([
+        [handler, context, Boolean(once)]
+      ]);
     },
 
-    off: function (name, handler) {
+    off: function (name, handler, context) {
+      if (context === null)
+        context = undefined;
       if (!this.__bindings.hasOwnProperty(name))
         return;
-      var index = $.inArray(handler, this.__bindings[name]);
-      if (index !== -1)
-        this.__bindings[name].splice(index, 1);
+      this.__bindings[name] = this.__bindings[name].filter(function (listener) {
```

Comment:
**tldr:** Fine, I'll change it :wink: 

That said, even if I somewhat see your point, it makes little sense to me that we refuse to ask the users to provide a working ES5 environment, but ask them to use jQuery to provide the missing tools (especially when Core.js is much more used than jQuery according to NPM stats).

On a side note and as a reminder, the new fullscreen will assume an ES6 environment. Babel and Core.js are already used together to provide a perfect environment in every circonstance.

> I find quite problematic too to ask users (even if it is only Wisembly front teams) to uses Core.js to support this piece of code.

As I mentionned, Wisembly already uses Core.js since https://github.com/Wisembly/wisembly-front/commit/bd8fabce4a7a973d0dfb8fdc7cdd5cc9a4c3cf80, and Solid requires IE10+. We don't have anything to change on our codebase either case, the issue is purely theoretical.